package.json

```diff
@@ -19,7 +19,6 @@
 	"engines": {
 		"node": ">=18"
 	},
-	"sideEffects": false,
```

Comment:
This was done automatically by `npm install`. :thinking:

Reply:
Weird. It should not remove custom properties.

Reply:
Oh this is because it is already present in line `18`. That's a duplicate line.